docs/sbpy/units.rst

```diff
@@ -111,6 +115,8 @@ phase angle can be calculated:
   >>> cross_sec = np.pi * (radius)**2
   >>> ref = mag.to('1/sr', reflectance('V', cross_section=cross_sec))
 
+.. doctest-requires:: synphot
+
```

Comment:
Again, do we need to set up this doctest requires before line 86 and line 107 and line 143?

Reply:
Got it.